onebusaway-android/src/main/java/org/onebusaway/android/ui/TripDetailsListFragment.java

```diff
@@ -26,6 +26,7 @@
 import org.onebusaway.android.io.elements.ObaTripStatus;
 import org.onebusaway.android.io.request.ObaTripDetailsRequest;
 import org.onebusaway.android.io.request.ObaTripDetailsResponse;
+import org.onebusaway.android.tripservice.BootstrapService;
```

Comment:
Looks like this import can be removed - in Android Studio, when committing you can check "Optimize imports" to do this automatically.